### PR TITLE
AST-1215 - Build: Release PR v3.7.5

### PR DESCRIPTION
**Pull request by sagarb90**
### Description
- Final release PR. Please check all the checklist below before merging.
---

### Checklist:
- [x] Finalize changelog.txt
- [x] Add doc links in the changelog.txt
- [x] Change minimum addon update version - ASTRA_EXT_MIN_VER
- [x] Run to update packages        : `npm audit fix`
- [x] Run to produce minified files : `grunt minify`
- [x] Run to update Google Fonts    : `grunt google-fonts`
- [x] PHPCS - vendor/bin/phpcs
- [x] Run to update version number  : `grunt version-bump --ver=<version-number>`
- [x] Check the Background Process function version
- [x] Update - Tested Upto - style.css & readme.txt
- [x] Update Readme.txt file and then run: `grunt readme`
- [x] Generate POT file             : `wp i18n make-pot wp-content/themes/astra wp-content/themes/astra/languages/astra.pot`
- [ ] Add draft Release notes on github
- [ ] Add draft changelogs to wpastra.com
- [ ] Open PR to backport branch to `next-release`
---

### Description
Final release PR. Please check all the checklists below in the same sequence before merging.

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -25,7 +25,7 @@
  * Minimum Version requirement of the Astra Pro addon.
  * This constant will be used to display the notice asking user to update the Astra addon to the version defined below.
  */
-define( 'ASTRA_EXT_MIN_VER', '3.6.2' );
+define( 'ASTRA_EXT_MIN_VER', '3.6.3' );
```
Comment (imnavanath) [inline functions.php:28]: I think no need to make dependent update. Your thoughts @premanshup ?

**Review by premanshup — APPROVED** [review]

File: functions.php (hunk 1 above)
Comment (sagarb90) [reply functions.php:28]: I think we need this because some of the code related to  Elementor widgets are in addon as well.

Comment (premanshup) [reply functions.php:28]: @imnavanath Let's keep this.

Comment (imnavanath) [reply functions.php:28]: 👍

**Review by imnavanath — APPROVED** [review]

**Pull request merged**